lib/translation_io/config.rb

```diff
@@ -54,10 +60,11 @@ def initialize
       self.pot_package_version       = '1.0'
       self.pot_copyright_holder      = File.basename(Dir.pwd)
       self.pot_copyright_year        = Date.today.year
+      self.text_domain               = TEXT_DOMAIN
```

Comment:
Default this to 'app'
Probably should remove the old constant and just set this to a string here